akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -61,6 +61,9 @@ abstract class JournalSpec(config: Config)
     senderProbe = TestProbe()
     receiverProbe = TestProbe()
     preparePersistenceId(pid)
+  }
+
+  protected def writeDefaultMessages(): Unit = {
```

Comment:
Since this is the TCK there could be plugins that have added more tests to their subclass of JournalSpec and rely on that those default messages are initialized.

We could still extract it to `def writeDefaultMessages()`, but call it from `beforeEach` so that it works as before. Then we can override in the problematic test for leveldb.

Reply:
That was my naive way to attack it as well, the problem is that sine the JournalPerfSpec extends this test, it includes all these tests in addition to its own performance tests so then all the inherited tests fail. I also tried with some toggle to flip after the inherited tests were run but that was messy. 

I understand that a journal could add its own tests but I think that the journal plugin having to add an explicit message setup was the least invasive I could think of.

Optimally there would be an journal spec infra supertype that both could inherit but introducing that could also break journal tests.